pkg/util/traceevent/flightrecorder.go

```diff
@@ -120,6 +120,31 @@ type SuspiciousEventConfig struct {
 	// QueryFail error code?
 	// ResolveLock?
 	// RegionError
+	IsInternal bool            `json:"is_internal,omitempty"`
+	DevDebug   *DevDebugConfig `json:"dev_debug,omitempty"`
+}
+
+// DevDebugConfig is the configuration for development debugging.
+type DevDebugConfig struct {
+	Type string
+}
+
+// compile validates the development debugging configuration.
+func (c *DevDebugConfig) compile(b *strings.Builder, mapping *dumpTriggerConfigCompiled, conf *DumpTriggerConfig) (uint64, error) {
```

Comment:
`dumpTriggerConfigCompiled` sounds more like a flag, not type name

Reply:
done